src/sass/components/_footer.scss

```diff
@@ -110,8 +112,24 @@
       }
     }
     .footer__bottom-links {
-      grid-area: about_wrapper;
-
+      grid-area: contact_wrapper;
+
+      .footer__bottom-links--wrapper {
```

Comment:
Might be able to use a similar technique as `.footer__menu--list` if we can reorder the markup.

Reply:
This would require rewriting the css for the footer which will require more time. Since its already working, I dont like to change css that doesn't need it.